vendor/github.com/mongodb/mongo-tools-common/db/db.go

```diff
@@ -184,9 +183,6 @@ func configureClient(opts options.ToolOptions) (*mongo.Client, error) {
 	}
 	if opts.WriteConcern != nil {
 		clientopt.SetWriteConcern(opts.WriteConcern)
-	} else {
-		// If no write concern was specified, default to majority
-		clientopt.SetWriteConcern(writeconcern.New(writeconcern.WMajority()))
```

Comment:
If all the tools default to majority, then I think we need to keep this.

Reply:
Reverting this change, but before this PR is merged I will have to do more changes here to ensure URI options aren't ignored (once go driver bump happens).